server/routes/register.js

```diff
@@ -0,0 +1,43 @@
+import express from "express";
+import User from "./../models/User";
+var router = express.Router();
+
+
+router.post("/", async function(req, res, next) {
+	const email = req.body.email;
+	const password = req.body.password;
```

Comment:
we going to have a confirm password as well right? Are there any other fields like name coming in? We may need to create a profile in this route with same other details as well. 

This may be somethign other team members working on the front-end can comment about

Reply:
Okay, got it